RELEASE.md

```diff
@@ -0,0 +1,26 @@
+# Release Process
+
+## Overview
+
+At any given time there are three versions of the Prometheus charm
+available
+
+1. Stable - This is a well tested production ready version of the
```

Comment:
The fact that these are listed in reverse order wrt the next paragraph makes my OCD flare :D

Reply:
Fixed.